golang/cosmos/app/app.go

```diff
@@ -788,6 +767,41 @@ func NewAgoricApp(
 	return app
 }
 
+func upgrade8Handler(app *GaiaApp, upgradeName string) func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
```

Comment:
The `upgradeName` here shadows the constant of the same name - rename to `name` here, or rename the constant to `upgradeMainName` or something.